docs/evaluation/simulate.rst

```diff
@@ -58,39 +64,90 @@ want to skip this section.
+in a configuration close to what we want to simulate. This PHA file
+that already specifies the response and backgrounds so that they can be automatically
+loaded. Also, the exposure time, background scaling etc. will be taken
+from the header of the file, but we can change these values if we want.
+When new data is simulated later, the counts in ``data`` will be overwritten,
+but all other information stays the same.
+In this example, we use a datafile from Sherpa's test data files
+(which are large and not installed by default
+:ref:`but can be downloaded <install-sherpa-test-data>`).
+
+ .. plot::
+    :include-source:
+    :context:
+    :nofigs:
+
+    >>> from sherpa.utils.testing import get_datadir
+    >>> data_dir = get_datadir() + '/'
+    >>> from sherpa.astro.io import read_pha
+    >>> data = read_pha(data_dir + '9774_bg.pi')
+    >>> data.exposure = 5e5  # Simulate longer exposure than in the original data
+
+Alternatively, we first create a :py:class:`~sherpa.astro.data.DataPHA` object with
+the correct channel numbers, responses, and exposure time. In this example, we
+initially set the exposure time and then add the remaining information line by line:
+
+.. plot::
+    :include-source:
+    :context:
+    :nofigs:
+
+    >>> import numpy as np
+    >>> from sherpa.astro.io import read_arf, read_rmf
+    >>> from sherpa.astro.data import DataPHA
+    >>> data = DataPHA(name='any', channel=None, counts=None, exposure=10000.)
+    >>> data.set_arf(read_arf(data_dir + '9774.arf'))
+    >>> data.set_rmf(read_rmf(data_dir + '9774.rmf'))
+    >>> # By convention, channel numbers usually start at 1
+    >>> data.channel = np.arange(1, data.get_rmf().detchans + 1)
+    >>> data.counts = np.zeros_like(data.channel)
 
 Next, we set up a model. In this case, we start with a powerlaw source
 where the slope and normalization of that powerlaw are already known,
 e.g. from the literature. We then add a weak emission line. Our
 simulation will show us if this emission line would be detectable in
-a real observation::
-
-  >>> from sherpa.models.basic import PowLaw1D, Gauss1D
-  >>> pl = PowLaw1D()
-  >>> line = Gauss1D()
-  >>> pl.gamma = 1.8
-  >>> pl.ampl = 2e-05
-  >>> line.pos = 6.7
-  >>> line.ampl = .0003
-  >>> line.fwhm = .1
-  >>> srcmdl = pl + line
+a real observation:
+
+.. plot::
+    :include-source:
+    :context:
+    :nofigs:
+
+    >>> from sherpa.models.basic import PowLaw1D, Gauss1D
+    >>> pl = PowLaw1D()
+    >>> line = Gauss1D()
+    >>> pl.gamma = 1.8
+    >>> pl.ampl = 2e-05
+    >>> line.pos = 6.7
+    >>> line.ampl = .0003
+    >>> line.fwhm = .1
+    >>> srcmdl = pl + line
+
+We will need to wrap this model with the response (ARF and RMF), which takes care
+of converting from channels to energy:
+
+.. plot::
+    :include-source:
+    :context:
+    :nofigs:
+
+    >>> from sherpa.astro.instrument import Response1D
+    >>> resp = Response1D(data)
```

Comment:
We should be able to just say

```
>>> resp = data.get_full_response()
```

but I'm not sure if it's worth changing here.